src/style.css

```diff
@@ -1,3 +1,92 @@
-body {
+body, h1, h2, h3, p, ul {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
```

Comment:
The 'box-sizing: border-box;' property is applied only to the body. To apply it to all elements, consider using 'body, *, *::before, *::after { box-sizing: border-box; }'.